docs/SUMMARY.md

```diff
@@ -1,20 +1,21 @@
 # Summary
 
 - [Welcome](./welcome.md)
-- [Overview](./overview/README.md)
+- [Overview](./overview.md)
+- [Getting Started](./getting_started.md)
+    - [Rust](./libraries/rust/getting_started.md)
+    - [Node.js](./libraries/nodejs/getting_started.md)
+    - [Python](./libraries/python/getting_started.md)
```

Comment:
Java is missing here, do you only want to add it later when the getting_started.md is also there?

Reply:
Yeah, I would like to avoid PR creep.